src/gtest/test_metrics.cpp

```diff
@@ -107,3 +107,29 @@ TEST(Metrics, EstimateNetHeight) {
     }
     RegtestDeactivateBlossom();
 }
+
+TEST(Metrics, NextUpgrade) {
+    SelectParams(CBaseChainParams::REGTEST);
+    const Consensus::Params& params = Params().GetConsensus();
+
+    UpdateNetworkUpgradeParameters(Consensus::UPGRADE_TESTDUMMY, Consensus::NetworkUpgrade::ALWAYS_ACTIVE);
+    UpdateNetworkUpgradeParameters(Consensus::UPGRADE_TESTDUMMY, 100);
```

Comment:
Remove the redundant first line (these calls just set a single chainparams value, and the second one overwrites the first).